src/components/user/tour/create-tour/index.js

```diff
@@ -0,0 +1,162 @@
+import React, {Component} from 'react';
+import Button from "@material-ui/core/Button";
+import Modal from '@material-ui/core/Modal';
+import TourDetail from "./tour-detail";
+import Guest from "./guests";
+import "./create-tour.css";
+import {GUESTS_COLUMNS, GUESTS_ROWS} from "../utils/constants";
+
+const CREATE_TOUR_STEPS = [
+    {
+        component: TourDetail,
+        formKey: 'tourDetail'
+    },
+    {
+        component: Guest,
+        formKey: 'guest',
+        dataKey: 'guestsGridRows'
+    },
+];
+
+class CreateTour extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
```

Comment:
I think this component should be overwritten. For form handling please use Formik. Everything related to form inputs will be changed. Will discuss that functionality after changing to Formik.